### ISSUE TITLE
npm dependencies update sanity check

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/fsa-style#130]

**Comment by francisrupert** [comment]
```
 politespace          ^0.1.7  →  ^0.1.20
 autoprefixer         ^6.3.6  →   ^6.5.1
 cssnano              ^3.5.2  →   ^3.7.7
 grunt-build-control  ^0.7.0  →   ^0.7.1
 grunt-concurrent     ^2.2.1  →   ^2.3.1
 grunt-sass           ^1.1.0  →   ^1.2.1
 load-grunt-tasks     ^3.4.1  →   ^3.5.2
 node-sass            ^3.4.2  →  ^3.10.1
 pixrem               ^3.0.0  →   ^3.0.2
 time-grunt           ^1.3.0  →   ^1.4.0
```

**Comment by francisrupert** [comment]
closed via 	833ee9f,	9260344, 	e402fd1
